### DISCUSSION TITLE
Question: Is it possible to show svg image in NavigationViewItem?

### DISCUSSION THREAD

**Discussion by BorzillaR** [microsoft/microsoft-ui-xaml#7506] [Q&A]
Hello!

I want to show svg image in NavigationViewItem in my WinUI for UWP application.
Is it possible?

**Comment by YuliKl** [comment]
@BorzillaR Ideally, you're looking for SVG support in BitmapIcon, as proposed in #1494. Unfortunately that doesn't work yet. Instead, you can retemplate your NavigationViewItems to use an Image instead of an icon, and add SVGs that way. Not sure if that feels like the right amount of work for your scenario.

**Comment by BorzillaR** [comment]
@YuliKl, thank you!

**Comment by maxkatz6** [comment]
Also NavigationViewItem uses IconElement in the API, so it should be possible to define your own IconElement with svg support inside of its template or copy source of BitmapIcon. I haven't tested it, but should just work.
